python_modules/dagster/dagster/core/definitions/infos.py

```diff
@@ -57,36 +40,35 @@ def __new__(cls, context, inout_def, solid, expectation_def):
         )
 
 
-class TransformExecutionInfo(
-    namedtuple('_TransformExecutionInfo', 'context config step pipeline_def resources log')
-):
-    '''An instance of TransformExecutionInfo is passed every solid transform function.
+class ITransformExecutionInfo(six.with_metaclass(ABCMeta)):  # pylint: disable=no-init
```

Comment:
or `TransformExecutionInfoMeta`?

Reply:
i'll rename this in the other PR to avoid merge flog